### PR TITLE
Fix swap prop type errors

### PR DESCRIPTION
**Pull request by danjm**
A few prop type errors coming from swaps code. This PR corrects them.

### CODE REVIEW

**Review by workfilemmfloe — DISMISSED** [review]
All checks good.

File: ui/app/pages/swaps/index.js [hunk 1]
```diff
@@ -325,7 +325,7 @@ export default function Swap () {
                 return aggregatorMetadata
                   ? (
                     <LoadingQuote
-                      loadingComplete={!fetchingQuotes && Object.values(quotes).length}
+                      loadingComplete={!fetchingQuotes && Boolean(Object.values(quotes).length)}
```
Comment (Gudahtt) [inline ui/app/pages/swaps/index.js:328]: Ah good, this is the one I had seen

File: ui/app/pages/swaps/select-quote-popover/sort-list/sort-list.js [hunk 2]
```diff
@@ -171,6 +171,6 @@ SortList.propTypes = {
   quoteDataRows: PropTypes.arrayOf(QUOTE_DATA_ROWS_PROPTYPES_SHAPE).isRequired,
   sortDirection: PropTypes.number.isRequired,
   setSortDirection: PropTypes.func.isRequired,
-  sortColumn: PropTypes.string.isRequired,
+  sortColumn: PropTypes.oneOfType([null, PropTypes.string]).isRequired,
```
Comment (Gudahtt) [inline ui/app/pages/swaps/select-quote-popover/sort-list/sort-list.js:174]: ... does this work? :thinking: `null` isn't a type...

Comment (Gudahtt) [reply ui/app/pages/swaps/select-quote-popover/sort-list/sort-list.js:174]: One alternative would be making it not required, and setting a default value of `null` instead. That would get the same intended effect I think.

Comment (danjm) [reply ui/app/pages/swaps/select-quote-popover/sort-list/sort-list.js]: done in cf1480a9b

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Pull request merged**